ansible/ci-initial-setup.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Initial setup to enable administration with ansible.
+
+- hosts: all
```

Comment:
Removing it doesn't work, it tells me it's missing.  How do I restrict to just the specified host?  This is not intended to be run more than once on a single specified host; it's one-time setup to make that host manageable by ansible.